### PR TITLE
Fix for #74

### PR DESCRIPTION
**Pull request by tomjn**

### CODE REVIEW

**Comment by tomjn** [comment]
This doesn't fix the fake term stuff done elsewhere, to fix that just make sure it only happens on the classic editor screen

**Comment by tomjn** [comment]
and remember, the state determines the UI, not the other way around. You can destroy the DOM completely and recreate it from the state and React, here the HTML tags hold no value other than to inform the user.

For example, the radio inputs are not selected. They look selected, but they're just really fancy alternatives for modifying the data store, the data store is what matters. The radio buttons show the user what the data store says, and when you click on a different radio input, that new radio input only shows as selected because the data store says it is, not because the user selected it.

**Comment by helgatheviking** [comment]
I wanted to look at this today, but the build tools have defeated me. `npm run` will not run for me.

**Comment by tomjn** [comment]
I wrote this on github so it's possible there are errors

**Comment by helgatheviking** [comment]
pretty sure it's not you. it's the WordPress build scripts and my windows set up.

**Comment by helgatheviking** [comment]
Oh, might also be you. build tools are finally building! But seeing this
>Uncaught ReferenceError: terms is not defined

File: js/src/radio-term-selector.js [hunk 1]
```diff
@@ -405,6 +412,18 @@ class RadioTermSelector extends Component {
 				role="group"
 				aria-label={ groupLabel }
 			>
+				<div className={ 'editor-post-taxonomies__' + klass + '-terms-choice ' }>
+					<input
+						id={ `editor-post-taxonomies-${ klass }-term-${ -1 }` }
+						className={ 'editor-post-taxonomies__' + klass + '-terms-input components-radio-control__input' }
+						type="radio" // @helgatheviking
+						checked={ terms.length === 0 }
```
Comment (tomjn) [inline js/src/radio-term-selector.js:420]: @helgatheviking this needs replacing with a check to see that there are no terms assigned in this taxonomy to the current post. Retrieving `terms` in this component will do the trick

Comment (helgatheviking) [reply js/src/radio-term-selector.js:420]: I think I already have that covered [here](https://github.com/helgatheviking/Radio-Buttons-for-Taxonomies/blob/master/js/src/radio-term-selector.js#L319). I think what I might try is to add your extra link into the component and see how I get along!

**Comment by helgatheviking** [comment]
Closing in favor of #84

**Pull request closed without merging**